SimPEG/EM/TDEM/ProblemTDEM.py

```diff
@@ -650,10 +647,170 @@ class Problem3D_e(BaseTDEMProblem):
     _formulation = 'EB'
     fieldsPair = Fields3D_e  #: A Fields3D_e
     surveyPair = SurveyTDEM
+    Adcinv = None
 
     def __init__(self, mesh, **kwargs):
         BaseTDEMProblem.__init__(self, mesh, **kwargs)
 
+    def Jtvec(self, m, v, f=None):
```

Comment:
Hmm... there must be a way, but currently, I prefer having separate Jtvec, but let me know if you have a smart way to do this..